packages/tradershub/src/features/cfd/screens/Jurisdiction/JurisdictionCard/JurisdictionCardBack.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { Dispatch, FC, SetStateAction } from 'react';
+import { Text } from '@deriv/quill-design';
+import BackArrowIcon from '../../../../../public/images/ic-back-arrow.svg';
+import { verificationIconsMapper, verificationStatusIconsMapper } from '../constants';
+import { jurisdictionVerificationContents } from '../jurisdiction-contents/jurisdiction-verification-contents';
+import {
+    TJurisdictionCardItemVerification,
+    TJurisdictionCardItemVerificationItem,
+} from '../jurisdiction-contents/props.types';
+
+type TJurisdictionCardBackProps = {
+    setIsFlipped: Dispatch<SetStateAction<boolean>>;
+    verificationDocs?: TJurisdictionCardItemVerification;
+};
+
+const JurisdictionCardBack: FC<TJurisdictionCardBackProps> = ({ setIsFlipped, verificationDocs }) => {
+    const verificationContents = jurisdictionVerificationContents();
+    if (verificationDocs)
+        return (
```

Comment:
Just a suggestion for better code readability! 
```suggestion
    if (!verificationDocs) <></>;
    return (
```

Reply:
I was thinking the same but i tend to use the Wallets approach thats why i didn't change.
But i agree with this so lets go with this